kex.c

```diff
@@ -154,6 +154,18 @@ kex_alg_list(char sep)
 	return ret;
 }
 
+char *
+kex_alg_list(char sep)
+{
+	return kex_alg_list_internal(sep, kexalgs);
+}
+
+char *
+kex_gss_alg_list(char sep)
```

Comment:
I think this should be `gss_kex_alg_list`, since `gss_kex_names_valid` already exists in this file with the same prefix.

Reply:
The question is whether the `gss_kex_names_valid` should not logically be `kex_gss_names_valid` since it is somehow in the scope of the kex "module".

Reply:
I'd be OK with that; I just want the two to be consistent.